lib/js/src/compose/types/page-block/geometry/feed.ts

```diff
@@ -53,4 +53,8 @@ export default class Feed {
       }
     }
   }
+
+  isValid (): boolean {
+    return this.options.moduleID !== NoID && this.geometryField !== ''
```

Comment:
https://github.com/cortezaproject/corteza/blob/8a7a10edaccb4b9d271ec4e100e22201c5c3e27a/lib/js/src/compose/types/page-block/geometry/feed.ts#L10

I noticed that the geometryField options is not updated when a field is not selected. My question is should it even be in the options? It either looks like we have two identical things and we only really need one or we are not updating it as we should. Or maybe I am the one missing something